### PR TITLE
Vodafone testing preparation

### PR DESCRIPTION
**Pull request by alexAlice**

### CODE REVIEW

File: packages/donations-app/app/components/home/homeController.js [hunk 1]
```diff
@@ -1,19 +1,43 @@
...
+    }
 
-          return _.pick(project, keyList);
-        });
+    // FIXME
+    // Livia asked to move fusion-housing project from the first position of
+    // active projects
+    // The problem is connected with the fact that fusion housing project was
+    // the second project created on stage (after st-mungos)
+    function doHackForFusionHousing(projects) {
```
Comment (jwojciechowski) [inline packages/donations-app/app/components/home/homeController.js]: It'll be better to add an order field to the project, similar to the improvement planned for outcomes.
It's not urgent, but if we want to postpone it, let's add a jira issue for it.

File: packages/donations-app/app/components/home/homeController.js [hunk 2]
```diff
@@ -1,19 +1,43 @@
...
     var loadData = function () {
       vm.contact = {};
       loadProjectsWithCharities();
     };
 
-    var loadProjectsWithCharities = function () {
-      ProjectService.getActiveProjects().then(function (projects) {
-        vm.projects = _.map(projects.data, (project) => {
-          let keyList = ['title', 'code', 'charity', 'lead', 'img'];
+    if (MODE != 'prod') {
```
Comment (jwojciechowski) [inline packages/donations-app/app/components/home/homeController.js]: Which mode do we want to use for Vodafone, aren't we going to accept real money?

File: packages/donations-app/app/components/home/homeView.html [hunk 3]
```diff
@@ -55,6 +55,23 @@ <h3 class="uppercase">
...
+  <section class="container"
+            id="home-completed-projects">
+     <div class="row">
+       <h3 class="uppercase">
+         Visit our completed projects
+       </h3>
+     </div>
+     <div class="row">
+         <homepage-project-card ng-repeat="project in homeCtrl.projects"
+                                ng-if="project.status == 'FINISHED'"
```
Comment (jwojciechowski) [inline packages/donations-app/app/components/home/homeView.html]: let's also filter out mocked projects here

File: packages/donations-app/app/components/project-wizard/projectWizardOutcomeView.html [hunk 4]
```diff
@@ -117,6 +117,26 @@
...
+
+    <div class="col-sm-9">
+      <input type="text" class="form-control" ng-model="outcome.actionVerbForProgress"
+             placeholder="Outcome action verb for progress, example: help"
+             name="actionVerbForProgress" required>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <label class="col-sm-3 control-label">Unit for action verb</label>
```
Comment (jwojciechowski) [inline packages/donations-app/app/components/project-wizard/projectWizardOutcomeView.html]: Let's put a more verbose description with an example ("A part of text displayed on my impact splash beside "We raised $100 to [help]...", it's easy to forget what is the true meaning of the parameter.

File: packages/donations-app/app/components/home/homeController.js (hunk 2 above)
Comment (alexAlice) [reply packages/donations-app/app/components/home/homeController.js]: Livia said, we are going to use stage

File: packages/donations-app/app/components/home/homeController.js (hunk 1 above)
Comment (alexAlice) [reply packages/donations-app/app/components/home/homeController.js]: I have created JIRA task https://alice-si.atlassian.net/browse/AD-394

**Pull request merged**